practices/done.md

```diff
@@ -0,0 +1,66 @@
+# Guiding Principles
+
+The customer can use our work to solve their problem / the work
+delivers (the negotiated) value.
+
+# DCE Definition of Done
+
+1.  Code is peer-reviewed
+2.  Other work (as appropriate) is peer-reviewed
+3.  Code is deployed to test environment
+4.  Feature includes automated test(s) against acceptance
+    criteria
+5.  Feature passes regression testing
+6.  (aspiration) Feature passes smoke test
+7.  (aspiration) Feature is documented sufficiently and/or
+    appropriately (the need for special knowledge is minimized) &
+    documentation is minimized
+8.  Feature ok-ed by Product Owner
+
+# Some readings on the topic
+
+[Definition of Ready](https://manifesto.co.uk/the-definition-of-ready/)
+
+The Definition of Ready is a Scrum team's shared understanding of what criteria a story needs to meet before it can be taken on.
+
+[Definition of Done](https://manifesto.co.uk/definition-done/)
+
+The Definition of Done is a Scrum team's shared understanding of which
+criteria a story must satisfy to make it into a releasable increment of
+the product. (edited) 
+
+# The Agile Samurai
+
```

Comment:
@little9 Would you mind nesting these headings? I would think that all of the individual readings or quotes should nest underneath "Some readings on the topic" instead of displaying as top level headings.

Reply:
No problem

Reply:
:+1: